src/main/java/com/microtripit/mandrillapp/lutung/http/MandrillService.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * 
+ */
+package com.microtripit.mandrillapp.lutung.http;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import com.microtripit.mandrillapp.lutung.model.MandrillApiError;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * @author rschreijer
+ * @since Mar 19, 2013
+ */
+public final class MandrillService {
```

Comment:
very good that now we have a MandrillService, however making it final is a bit of an overkill imo.